azure-pipelines.yml

```diff
@@ -42,11 +42,11 @@ jobs:
   - template: /eng/performance/benchmark_jobs.yml
     parameters:
       osName: windows
-      osVersion: RS4
+      osVersion: RS5
       kind: micro
       architecture: x64
       pool: Hosted VS2017
-      queue: Windows.10.Amd64.ClientRS4.Open
+      queue: Windows.10.Amd64.ClientRS5.Open
```

Comment:
I guess we need the new queue names and to update this. (in several spots.)

Reply:
Yes. I am going to wait until it is rolled out and the machines have been updated to make that change.

Reply:
Made the change. This can't be checked in until the new rollout to create the queue and the machines have been moved to the new queue name.